### DISCUSSION TITLE
Randomly Refresh News Articles in 'My Feed' section

### DISCUSSION THREAD

**Discussion by marcocabrerahdez** [dailydotdev/daily#1134] [Feature Requests]
**Description:**
Having a static news feed can sometimes limit exposure to diverse content. To enhance user engagement and content discoverability, I propose the addition of a feature that allows for random refreshes of news articles in the "My Feed" section.

**Feature Suggestion:**
Implement a button or enable automatic refreshing functionality that randomly populates new news articles each time the My Feed section is either manually refreshed by the user or automatically upon certain intervals.

**Benefits:**
1. Enhanced Engagement: Users will likely engage more frequently with the platform as they can discover new content each time they refresh.
2. Increased Discoverability: Helps in promoting a wider range of articles, which might otherwise go unnoticed.
3. User Satisfaction: Keeps the feed dynamic and interesting, improving the overall user experience.

> This feature could be implemented as a toggle option in the settings, allowing users to choose how often they want their feed to be refreshed or if they prefer a static feed.

**Comment by idoshamun** [comment]
Hi @marcocabrerahdez,

We tried to add this button in the past but it had negative impact. We may consider trying it again at some point but it will take some time
